clients/src/main/java/org/apache/kafka/common/errors/OffsetOutOfRangeExceptionWithOffsetValues.java

```diff
@@ -0,0 +1,26 @@
+package org.apache.kafka.common.errors;
```

Comment:
Adding exception types to this package requires a KIP. I would suggest moving this to the broker. Also, we are missing the license.

Reply:
Move the class definition into ReplicaManager